riscv-sbi.adoc

```diff
@@ -257,6 +290,291 @@ call doesn't return.
 | *RESERVED*                |             |    0x09-0x0F |
 |===
 
+
+== Timer Extension, Extension ID: 0x54494D45 (TIME)
+This replaces legacy timer extension (0x00). It follows the new calling convention
+defined in v0.2.
+
+[source, C]
+----
+struct sbiret sbi_set_timer(uint64_t stime_value)
+----
+Programs the clock for next event after *stime_value* time. *stime_value* is in absolute
+time. This function must clear the pending timer interrupt bit as well.
+
+If the supervisor wishes to clear the timer interrupt without scheduling the next
+timer event, it can either request a timer interrupt infinitely far into the
+future (i.e., (uint64_t)-1), or it can instead mask the timer interrupt by
+clearing sie.STIE.
+
+== IPI Extension, Extension ID: 0x735049 (IPI)
+This extension replaces the legacy extension (0x03). All the functions follow
+the `hart_mask` defined in binary encoding section.
+
+[source, C]
+----
+struct sbiret sbi_send_ipi(const unsigned long hart_mask, unsigned long hart_mask_base)
```

Comment:
The `const` here doesn't make any sense.

Reply:
Yeah. I will fix it.